server.go

```diff
@@ -133,52 +133,121 @@ func (app *Yarana) replyText(replyToken, text string) error {
 	return nil
 }
 
-func (app *Yarana) handleText(message *linebot.TextMessage, replyToken string, source *linebot.EventSource) error {
-	// Handle user input
-	switch message.Text {
+func (app *Yarana) handleText(message *linebot.TextMessage, replyToken string, source *linebot.EventSource) (err error) {
+	// Analyze text message
+	userReq := NewUserTextRequest()
+	err = userReq.AnalyzeInputText(message.Text)
+	if err != nil {
+		app.sorryAndShowHelp(replyToken) // TODO: Do we have to manange replyToken? because we can use replyToken only once in a request.
```

Comment:
We have to consider behavior of reply message when the bot got incomprehensible message.